OpenEdXMobile/res/values/strings.xml

```diff
@@ -463,8 +463,6 @@
     <string name="facebook_text">Facebook</string>
     <!-- Google login method -->
     <string name="google_text">Google</string>
-    <!-- Message shown when attempting to find courses with no Internet connection -->
-    <string name="find_courses_offline_text">You are in offline mode.\nTo find courses, connect to the Internet.</string>
```

Comment:
@saeedbashir confirmed that this is an unused string on iOS too, so we are safe to remove this from our repo as well.